vip_spec.xsd

```diff
@@ -348,7 +348,62 @@
 
 //-->
 
+<!--
 
+langString: a type representing a string with a language specified.
+
+-->
+<xs:complexType name="langString">
+    <xs:simpleContent>
+        <xs:extension base="xs:string">
+            <xs:attribute name="lang" type="xs:language" use="required"/>
+        </xs:extension>
+    </xs:simpleContent>
+</xs:complexType>
+<!--
+
+interText (short for "internationalized text"): a type representing a block
+  of text translated into one or more languages.
+
+For example, an instance of:
+
+    <xs:element name="office" type="interText"/>
+
+could be:
+
+    <office inter_text_id="office_mayor">
+        <text lang="en">Mayor</text>
+        <text lang="es">Alcalde</text>
+        <text lang="zh">市長</text>
+    </office>
+
+or (e.g. if only English is available):
+
+    <office>Mayor</office>
+
+The optional "inter_text_id" attribute is useful for tracking the string
+source, for example if the translated values came from localized resource
+files.
+
+-->
+<xs:complexType name="interText">
```

Comment:
This looks good, but I think `interText` is a bit vague without the descriptive context. I don't have an immediate suggestion, aside from spelling it out. @jdmgoogle @pstenbjorn any ideas?